lang/java/ipc/src/main/velocity/org/apache/avro/ipc/stats/templates/statsview.vm

```diff
@@ -51,7 +51,7 @@
 <title>$title</title>
 <script type="text/javascript" src="static/protovis-r3.2.js"></script>
 <script type="text/javascript" src="static/tipsy.js"></script>
-<script src="static/jquery-1.4.2.min.js" type="text/javascript"></script>
```

Comment:
Shouldn't we pull these resources from a CDN? https://code.jquery.com/jquery-1.6.3.min.js

Reply:
IMO, no. <https://httptoolkit.com/blog/public-cdn-risks/>